Update verified publisher icon (#4465)
dart-lang/site-www: src/tools/pub/verified-publishers.md

description: Learn what verified publishers are, and they're verified.
---

> The pub.dev verified publisher badge <img src="/assets/img/verified-publisher.svg" width="20" height="20" alt="pub.dev verified publisher logo"> 
> lets you know that a package
> was published by a publisher whose identity has been verified.

Reviewer: **issue (non-blocking):** Prefer that this run-on sentence with passive voice be fixed at the same time.
```suggestion
The pub.dev verified publisher badge <img src="/assets/img/verified-publisher.svg" width="20" height="20" alt="pub.dev verified publisher logo"> 
lets you know that the Dart team verified the identity of the developer who published this package.
```

Author: Sure. I changed Dart team to pub.dev as it is not manual verification but rather done automatically tied to DNS records.
